### ISSUE TITLE
"Future" techniques for several SC have never been developed

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#4763]
When I read through WCAG's sufficient techniques, there are 31 "techniques" marked as either "(future link)" (SC 1.3.6 Identify Purpose, 2.1.1 Keyboard, 4.1.3 Status Messages) or  as "(potential future link)" (SCs 1.4.10 Reflow, 1.4.13 Content on Hover or Focus, 2.5.2 Pointer Cancellation, 2.5.3 Label in Name, 2.5.6 Concurrent Input Mechanisms, 3.3.7 Redundant Entry, 3.3.8 Accessible Authentication, 3.3.9 Accessible Authentication (Enhanced)).  

It isn't clear why some are labeled "potential future links" vs. simply "future links" since no techniques have been forthcoming for quite some time. 

Is there any intent to develop these? If not, identifying them as a future link of any type isn't helpful. If no one is planning to work on techniques for these, perhaps these labels should be removed. Or perhaps it would be better to track the undeveloped techniques through a project plan with GitHub issues attached rather than mark them in the understanding documents in that way.

**Comment by fstrr** [comment]
There's a big list of these in #4378. A couple of people said they'd work on creating some but that unfortunately hasn't happened (yet?). That said, #4686 is up for AG review soon and that will close one of those "potential future links".

**Comment by mbgower** [comment]
@maryjom given the existence of #4378, I'm closing this  as a duplicate. Please reopen if you do not believe that is adequate.

**Comment by maryjom** [comment]
@mbgower Perhaps this issue should be about how such techniques that are not developed, but are being worked on, are identified. In looking at what techniques are "future", there are 5 ways they are identified:
1. Future link
2. Potential future link
3. Gx
4. GXXX
5. FXXX

All mean the same thing.  If there is a style guide for such things, it should be added to the style guide and consistently applied.
